src/com/owncloud/android/ui/adapter/LocalFileListAdapter.java

```diff
@@ -289,7 +291,8 @@ public void setSortOrder(Integer order, boolean isAscending) {
         FileStorageUtils.mSortOrderUpload = order;
         FileStorageUtils.mSortAscendingUpload = isAscending;
         if (mFiles != null && mFiles.length > 0) {
-            mFiles = FileStorageUtils.sortFolderUpload(mFiles);
+            mFiles = transformVecOCFile(FileStorageUtils.sortFolder(transformFileArray(mFiles),
```

Comment:
Same here. Besides, you could split the parameters into different lines to make it more readable:

```
mFiles = transformVecOCFile(
           FileStorageUtils.sortFolder(transformFileArray(mFiles),
           FileStorageUtils.mSortOrderUpload,
           FileStorageUtils.mSortAscendingUpload)
);
```